### DISCUSSION TITLE
Exclude from scan parameter

### DISCUSSION THREAD

**Discussion by ghost** [Qualys/log4jscanwin#41] [Ideas]
Would it possible to add a parameter to exclude certain disks\folders\file from scan? This could allow to avoid scan on huge data folders.

Thanks!

**Comment by romw** [comment]
See https://github.com/Qualys/log4jscanwin/issues/42

**Comment by romw** [comment]
Feature should be included as of:
https://github.com/Qualys/log4jscanwin/releases/tag/2.1.2.0
